src/test/scala/de/frosner/ddq/CheckTest.scala

```diff
@@ -296,7 +430,9 @@ class CheckTest extends FlatSpec with Matchers with BeforeAndAfterEach with Befo
     val df = makeIntegerDf(List(1,2,3), sql)
     val tableName = "myintegerdf1"
     df.registerTempTable(tableName)
-    Check.sqlTable(sql, tableName).hasNumRowsEqualTo(3).run shouldBe true
+    val constraint = Check.hasNumRowsEqualTo(3)
+    val result = ConstraintSuccess("The number of rows is equal to 3")
+    Check.sqlTable(sql, tableName).addConstraint(constraint).run().constraintResults shouldBe Map(constraint -> result)
```

Comment:
@Gerrrr Actually, this test can now be improved. It does no actually test the table name output so I will either add this to the `RunnerTest` or create a new integration test that checks if the table name is put as a header in the `CheckResult`